### PR TITLE
PR Template Improvements (Now Consistent With Bug Template)

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
Makes it clear where to write and, similarly to the issue report improvement of #40942 , removes the markup that hides your input (lol).

Comments appreciated :)

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
:eggplant:

**Review by Cyberboss — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -19,4 +21,11 @@ admin: messed with admin stuff
 server: something server ops should know
 /:cl:
 
-[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
+<!-- both :cl:'s are required for it to work! You can put your name beside the first :cl: if you want to overwrite your github username -->
```
Comment (subject217) [inline .github/PULL_REQUEST_TEMPLATE.md]: Capitalize both and github

Comment (ExcessiveUseOfCobblestone) [reply .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
<!-- Both :cl:'s are required for it to work! You can put your name beside the first :cl: if you want to overwrite your Github username -->
```

Comment (subject217) [reply .github/PULL_REQUEST_TEMPLATE.md]: i forgot to do it but there should be a period at the end of the sentence too please do it sweetie 😌

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -19,4 +21,11 @@ admin: messed with admin stuff
 server: something server ops should know
 /:cl:
 
-[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
+<!-- Both :cl:'s are required for it to work! You can put your name beside the first :cl: if you want to overwrite your Github username -->
```
[the remark this answers is not in the document]
Comment (ExcessiveUseOfCobblestone) [reply .github/PULL_REQUEST_TEMPLATE.md]: Please speak softly it's hard to write.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**